library/item/item_content_registry/src/main/java/org/quiltmc/qsl/item/content/registry/api/ItemContentRegistries.java

```diff
@@ -29,36 +29,36 @@
  * <p>
  * Current properties:
  * <ul>
- *     <li>{@link #FUEL_TIME}</li>
- *     <li>{@link #COMPOST_CHANCE}</li>
+ *     <li>{@link #FUEL_TIMES}</li>
+ *     <li>{@link #COMPOST_CHANCES}</li>
  * </ul>
  */
 public class ItemContentRegistries {
 	/**
 	 * The namespace for the content registries.
 	 */
-	public static final String NAMESPACE = "quilt_item_content_registry";
+	public static final String NAMESPACE = "quilt";
 
 	/**
 	 * A {@link RegistryEntryAttachment} for how long different items burn in a furnace. The value is stored in ticks.
 	 * <p>
-	 * Values can be set via code and through a data-pack with the file {@code data/quilt_item_content_registry/attachments/minecraft/item/fuel_time.json}
+	 * Values can be set via code and through a data-pack with the file {@code data/quilt/attachments/minecraft/item/fuel_times.json}
 	 */
-	public static final RegistryEntryAttachment<Item, Integer> FUEL_TIME = RegistryEntryAttachment
+	public static final RegistryEntryAttachment<Item, Integer> FUEL_TIMES = RegistryEntryAttachment
```

Comment:
Hm, I'm a bit wary of the plurification of the ICRs heres, although compared to the block ones? it actually kinda makes sense here? you'll need a second opinion though

Reply:
I think it makes sense.

Reply:
It makes sense.

Reply:
fair enough